ci-operator/step-registry/hypershift/agent/create/agent/hypershift-agent-create-agent-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ create agent command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted-service.tar.gz"
+
+# shellcheck disable=SC2087
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF |& sed -e 's/.*auths\{0,1\}".*/*** PULL_SECRET ***/g'
+
+# prepending each printed line with a timestamp
+exec > >(awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), \$0 }') 2>&1
+
+set -xeo pipefail
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
+REPO_DIR="/home/assisted-service"
+if [ ! -d "\${REPO_DIR}" ]; then
+  mkdir -p "\${REPO_DIR}"
+
+  echo "### Untar assisted-service code..."
+  tar -xzvf /root/assisted-service.tar.gz -C "\${REPO_DIR}"
+fi
+
+cd "\${REPO_DIR}/deploy/operator/"
+git clone https://github.com/LiangquanLi930/deployhypershift.git
```

Comment:
I guess you can move the `deploy_hypershift.sh` script you get from here to assisted-service / hypershift repo

Reply:
yes, when this PR is merged, I will move it